src/lucky_flow.cr

```diff
@@ -52,9 +74,9 @@ class LuckyFlow
 
   def take_screenshot(filename : String = generate_screenshot_filename, fullsize : Bool = true)
     if fullsize
-      with_fullsized_page { session.screenshot(filename) }
+      with_fullsized_page { driver.screenshot(filename) }
```

Comment:
Switched to the driver instead of the session so that the driver could make the screenshot directory if it's missing